samples/package.json

```diff
@@ -17,7 +17,7 @@
     "@google-cloud/logging-bunyan": "^0.9.5",
     "bunyan": "^1.8.12",
     "express": "^4.16.3",
-    "yargs": "^12.0.0"
+    "yargs": "^13.0.0"
```

Comment:
Does it make sense for this to be in devDependencies instead? </tangential>

Reply:
It's all fake money, since these modules never get published.  That having been said - yargs is required to run the sample itself, so I'd say regular deps sounds right.